src/Blockcore.Indexer.Core/Sync/SyncOperations.cs

```diff
@@ -214,17 +214,20 @@ private SyncBlockTransactionsOperation SyncBlockTransactions(IBlockchainClient c
 
          IEnumerable<Transaction> transactions = itemList.Select(s =>
          {
+            if (s.result == null)
+               return null;
```

Comment:
Perhaps have this in a .Where before the .Select

Reply:
fixed